data/rewards/init.go

```diff
@@ -24,6 +35,17 @@ type RewardMasterStore struct {
 	RewardCm *RewardCumulativeStore
 }
 
+type RewardMasterState struct {
```

Comment:
I was a little confused why we use a separate struct to dump the state. I'm guessing this is to wrap any other data we want to dump in the future?

Reply:
Yep. This is to wrap any other data from the stores we want to dump in the future, e.g. pending rewards.